eng/submit-source-build-release-pr.sh

```diff
@@ -21,13 +21,15 @@ a PR in the target repo."
    echo "--targetBranch, -b    (Optional) branch to submit the PR to, calculated automatically otherwise"
    echo "--globalJson, -g      (Optional) path to the global.json file to update"
    echo "--versionsProps, -p   (Optional) path to the Versions.props file to update"
+   echo "--resolvedSourceBuiltArtifactsFileName, -a   (Optional) actul Source Built Artifacts File Name"
+   echo "--resolvedSdkArtifactFileName, -s   (Optional) actul Sdk Artifact File Name"
```

Comment:
I'd maybe drop the `resolved` part? I am not sure what that means in this context